README.md

```diff
@@ -53,7 +53,8 @@ var foods = [ ["apple","banana","strawberry"], ["pizza","fries","hamburger"] ];
 How would I go about accessing the string "pizza" in the above array?
 
 ```js
-// write code here
+// write ce here
+foods[][1]
 ```
```

Comment:
`foods[0][1]`. You need to indicate you're accessing the first item in the `foods` array with `[0]`.